src/Form/AdminSettingsForm.php

```diff
@@ -644,6 +654,7 @@ private function setVisibilityValues(Config $settings, array $values) {
    */
   private function setAdvancedValues(Config $settings, array $values) {
     $settings->set('advanced.content_replacement_mode', $values['content_replacement_mode']);
+    $settings->set('advanced.bootstrap_mode', $values['bootstrap_mode']);
```

Comment:
Just me nitpicking again: please rotate line 656 and 657. Or not. I just wanted to point out any inconsistencies. ^^